components/script/dom/cssstylesheet.rs

```diff
@@ -100,6 +101,14 @@ impl CSSStyleSheet {
     pub fn set_origin_clean(&self, origin_clean: bool) {
         self.origin_clean.set(origin_clean);
     }
+
+    pub fn medialist(&self) -> DomRoot<MediaList> {
+        return MediaList::new(
+            self.global().as_window(),
+            self,
+            self.style_stylesheet().media.clone(),
+        );
```

Comment:
You can return it directly without specifying `return` explicitly in this case.

```suggestion
        MediaList::new(
            self.global().as_window(),
            self,
            self.style_stylesheet().media.clone(),
        )
```

Reply:
You're right. Fixed